Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleWindow.cpp

```diff
@@ -39,6 +39,12 @@ namespace ShaderManagementConsole
         m_documentInspector->SetDocumentId(documentId);
     }
 
+    void ShaderManagementConsoleWindow::CreateMenus(QMenuBar* menuBar)
+    {
+        Base::CreateMenus(menuBar);
+        m_actionSaveAsChild->setVisible(false);
+    }
+
```

Comment:
```suggestion
    void ShaderManagementConsoleWindow::UpdateMenus(QMenuBar* menuBar)
    {
        Base::UpdateMenus(menuBar);
        m_actionSaveAsChild->setVisible(false);
    }
```

use the update menus function to make sure that the your changes always override anything the base class might do